### PR TITLE
Add Liechtenstien holidays and tests.

### LINKED ISSUE #650: Add Holidays for Liechtenstien (country)
It would be great if you could add Liechtenstien.

      # Holidays Every Year same date 2022
      - '2022-01-01' #Neujahrstag
      - '2022-01-02' #Berchtoldstag
      - '2022-01-06' #Heilige Drei Konige
      - '2022-02-02' #Lichtmess
      - '2022-03-19' #Joseftag
      - '2022-05-01' #Tag der Arbeit
      - '2022-08-15' #Furstenfest (Staatsfeiertag)
      - '2022-09-08' #Maria Geburt
      - '2022-11-01' #Allerheiligen
      - '2022-12-08' #Maria Empfagnis
      - '2022-12-24' #Heiligabned
      - '2022-12-25' #1. Weihnachtstag
      - '2022-12-26' #2. Weihnachtstag
      - '2022-12-31' #Silvester
      # Holidays 2022
      - '2022-03-01' #Fasnachtsdienstag - https://en.wikipedia.org/wiki/Shrove_Tuesday
      - '2022-04-15' #Karfreitag
      - '2022-04-17' #Ostersonntag
      - '2022-04-18' #Ostermontag
      - '2022-05-26' #Christi Himmelfahrt
      - '2022-06-05' #Pfingstsonntag
      - '2022-06-06' #Pfingstmontag
      - '2022-06-16' #Fronleichnam

### PR DESCRIPTION
**Pull request by kasya**
Resolves #650

### CODE REVIEW

File: holidays/countries/liechtenstien.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+from dateutil.relativedelta import relativedelta as rd
+from holidays.constants import AUG, DEC, FEB, JAN, MAR, MAY, NOV, SEP
+from holidays.holiday_base import HolidayBase
+
+
+class Liechtenstien(HolidayBase):
+    """
+    Liechtenstien holidays.
+    See https://en.wikipedia.org/wiki/Public_holidays_in_Liechtenstein
+                for details.
```
Comment (arkid15r) [inline holidays/countries/liechtenstien.py:25]:
```suggestion
    Liechtenstien holidays.
    See https://en.wikipedia.org/wiki/Public_holidays_in_Liechtenstein
    for details.
```

File: holidays/countries/liechtenstien.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        # Saint Berchtold's Day.
+        self[date(year, JAN, 2)] = "Berchtoldstag"
+
+        # Epiphany.
+        self[date(year, JAN, 6)] = "Drei Könige"
+
+        # Candlemas.
+        self[date(year, FEB, 2)] = "Mariä Lichtmess"
+
+        # Shrove Tuesday
```
Comment (arkid15r) [inline holidays/countries/liechtenstien.py:43]:
```suggestion
        # Shrove Tuesday.
```

File: test/countries/test_liechtenstein.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    def setUp(self):
+        self.holidays = holidays.LI(observed=False)
+
+    def test_new_years(self):
+        for dt in (date(2010, 12, 30), date(2017, 1, 3)):
+            self.assertNotIn(dt, self.holidays)
+
+        for dt in (date(2022, 1, 1), date(2021, 1, 1)):
+            self.assertIn(dt, self.holidays)
+            self.assertEqual(self.holidays[dt], "Neujahr")
```
Comment (arkid15r) [inline test/countries/test_liechtenstein.py:29]:
```suggestion
        for year in (2021, 2022):
            dt = date(year, 1, 1),
            self.assertIn(dt, self.holidays)
            self.assertEqual(self.holidays[dt], "Neujahr")
```

**Review by arkid15r — APPROVED** [review]
LGTM

**Comment by dr-prodigy** [comment]
Merged, thanks! 👍

**Pull request merged**